lib/guardian/token/verify.ex

```diff
@@ -44,7 +44,7 @@ defmodule Guardian.Token.Verify do
   configured for the provided module
 
   Allowed drift is measured in seconds and represents the maximum amount
-  of time a token may be expired for an still be considered valid.
+  Gof time a token may be expired for an still be considered valid.
```

Comment:
@jsangilve could you remove this `G` that slipped in? Otherwise this is good IMO.